### PR TITLE
Disable IE canaries against PR

### PR DESCRIPTION
**Pull request by Stojdza**
#### Changes proposed in this Pull Request

Based on the discussion in p6fDka-CO-p2 reduce the number of IE canary runs:
- Disable IE canary run against PR
- Reduce scheduled IE canary run to once per day

#### Testing instructions

Make sure that IE canaries are not triggered on PR.

### CODE REVIEW

**Comment by blowery** [comment]
What's the motivation here?

**Comment by Stojdza** [comment]
@blowery Here is the previous discussion on this - p6fDka-CO-p2

File: .circleci/config.yml [hunk 1]
```diff
@@ -747,7 +747,7 @@ workflows:
           slack-webhook: '$SLACK_WOO'
     triggers:
       - schedule:
-          cron: '0 12,20 * * *'
+          cron: '0 12 * * *'
```
Comment (Stojdza) [inline .circleci/config.yml]: @bsessions85 I'm not sure about this one - this change will also affect Woo canaries. Should I split jobs, so Woo remains at two runs per day?

Comment (bsessions85) [reply .circleci/config.yml]: I would just run woo the same. It is still running on canary anyway

**Review by bsessions85 — APPROVED** [review]
This looks fine. Lets hold off a bit though as there is more discussion as to whether this will have any effect

**Comment by Stojdza** [comment]
Sure. Thanks @bsessions85!

**Comment by bsessions85** [comment]
Merging to get this out. We can revert later if they change their minds

**Pull request merged**